README.md

```diff
@@ -119,20 +138,53 @@ PunktF searches for the source path in the following order:
 
 ## Template Format
 
-### Replacement
+### Comments
+
+Comments can be inserted with `{{!-- ... --}}`. They will be ignored by the template
+parser and will not be transferred to the output.
+
+Example:
+
+```python
+{{!-- Inserts the current os name and prints it when executed --}}
+print("{{OS}}")
+```
+
+### Escaped
+
+If `{{` or `}}` need to used outside of a template block, put them inside an
+escaped block. Everything within it will get copied over without modification.
 
-Prefix (can be combined: e.g. {{#$RUSTC_PATH}}):
+Example:
+
+```text
+{{{ This is escaped ... I can use {{ without worry. I can even use }} and is still fine }}}
+```
+
+### Variables
+
+Prefix to determine where variables are looked for (can be combined: e.g. {{#$RUSTC_PATH}}):
+
+- None: First profile.variables then profile.file.variables
+- `$`: Only (system) ENVIRONMENT
+- `#`: Only profile.variables
+- `&`: Only profile.item.variables
 
-- None: First profile.env then profile.file.env
-- `$`: Only ENVIRONMENT
-- `#`: Only env
-- `&`: Only file.env
+
+Example:
 
 ```python
 rustc = {{RUSTC_PATH}}
 ```
 
-### Conditionals (TODO: think about structure)
+### Conditionals
+
+Supported are only if expressions with the following structure:
+
+`{{VAR}} (==|!=) "LITERAL"`
```

Comment:
can we also check if a variable exists or is defined? might be a neat feature. maybe using `var == NULL` or just `if VAR`.
If that's not already possible, please create a new issue or implement it here

Reply:
Sound like a good idea. Will do :+1:

Reply:
Done with e7b809373dbc94c1656d7afdf61cd6ebf2de261d